kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/KopServerStats.java

```diff
@@ -92,4 +92,10 @@
     String KOP_EVENT_QUEUE_SIZE = "KOP_EVENT_QUEUE_SIZE";
     String KOP_EVENT_QUEUED_LATENCY = "KOP_EVENT_QUEUED_LATENCY";
     String KOP_EVENT_LATENCY = "KOP_EVENT_LATENCY";
+
+    /**
+     * Network stats.
+     */
+    String KOP_NETWORK_TOTAL_BYTES_IN = "KOP_NETWORK_TOTAL_BYTES_IN";
+    String KOP_NETWORK_TOTAL_BYTES_OUT = "KOP_NETWORK_TOTAL_BYTES_OUT";
```

Comment:
Maybe we can remove the `KOP_` prefix in the string. Because in prometheus's metric string, the field already starts with `kop_server_` prefix. In addition, please also add the related verifications to `testMetricsProvider`.

Reply:
you are right !

My eyes weren't seeing the prefix !

I have updated the name, and I am adding tests, thanks